### PR TITLE
Feature/ivhms

### PR DESCRIPTION
**Pull request by YannikH**
**When merged this pull request will:**
- Adds an IVHMS MFD page which can be accessed through the action menu or a hidden button in the PFD page

### CODE REVIEW

File: addons/uh60_mfd/config/cfgVehicles.hpp [hunk 1]
```diff
@@ -14,8 +14,25 @@ class CfgVehicles {
...
+				displayName = "Open IVHMS";
+				displayNameDefault = "Open IVHMS";
+				priority = 0;
+				radius = 10; // A too small radius might cause the action to not be visible
+				position = "camera";
+				showWindow = 0;
+				hideOnUse = 1;
+				onlyForPlayer = 0;
+				shortcut = "";
+				condition = "player == driver this"; // Only show if the unit is alive and is not a player
```
Comment (ampersand38) [inline addons/uh60_mfd/config/cfgVehicles.hpp:32]: pilot only with this condition

File: addons/uh60_mfd/config/MFD/pages/monospace.hpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
+borderLeft=0;
+borderRight=0;
+borderTop=0;
+borderBottom=0;
+color[]=common_white;
+enableParallax=0;
+font="EtelkaMonospacePro";
```
Comment (ampersand38) [inline addons/uh60_mfd/config/MFD/pages/monospace.hpp:7]: I assume this font is messing up the jvmf spacing? https://cdn.discordapp.com/attachments/811711710666555422/920686026488430642/unknown.png

**Review by ampersand38 — CHANGES_REQUESTED** [review]

File: addons/uh60_mfd/config/cfgVehicles.hpp (hunk 1 above)
Comment (YannikH) [reply addons/uh60_mfd/config/cfgVehicles.hpp:32]: intentional

File: addons/uh60_mfd/config/MFD/pages/monospace.hpp (hunk 2 above)
Comment (YannikH) [reply addons/uh60_mfd/config/MFD/pages/monospace.hpp:7]: fixed

**Review by ampersand38 — APPROVED** [review]

**Comment by ampersand38** [comment]
Very neat!

**Pull request merged**